theano/gof/vm.py

```diff
@@ -705,6 +705,8 @@ def __init__(self, allow_gc=None, use_cloop=False, callback=None,
             version. If lazy is True or False, we force the version used
             between Loop/LoopGC and Stack.
 
+        c_thunks - If None or True, don't change the default. If False,
+            don't compile c code for the thunks.
         """
         # Note: if more parameters are added to __init__, make sure to forward
         # them in the "type(self)(...)" call in the "accept" method below.
```

Comment:
As this comment says, you have to add "c_thunks" to the list of keywords passed to `type(self)(...)` at line 754.